content/posts/GSoCStudents.md

```diff
@@ -0,0 +1,30 @@
+Congratulations to the 6 students selected to work with CircuitVerse during Google Summer of Code 2021! We’re thrilled to welcome them as a part of our community and we're looking forward to working together with our mentors and the greater CircuitVerse community 🎉 CircuitVerse's GSoC journey started from **3 students in 2019 to 4 in 2020 and finally 6 in 2021**! It has been nothing short of incredible 🚀
+
+Here a glimpse of our selected students:
+
+1. [Ruturaj Mohite](https://github.com/gr455) will be working on [Refactoring Testbench and Automated Testing](https://github.com/CircuitVerse/CircuitVerse/wiki/GSoC%2721-Project-List#project-1---testbench-project) and will be mentored by [Satvik Ramaprasad](https://github.com/satu0king/) and [Samiran Konwar](http://github.com/abstrekt/).
+
+2. [Pawan Joshi](https://github.com/pavanjoshi914) will be working on [Internationalization of CircuitVerse Platforms](https://github.com/CircuitVerse/CircuitVerse/wiki/GSoC%2721-Project-List#project-3---internationalization-i18n) and will be mentored by [Aboobacker MK](https://github.com/tachyons) and [Nitish Aggarwal](https://github.com/Nitish145).
+
+3. [Ayan Biswas](https://github.com/ayan-biswas0412) will be working on [Integration of CircuitVerse with LMS Platforms](https://github.com/CircuitVerse/CircuitVerse/wiki/GSoC%2721-Project-List#project-4---lms-integration) and will be mentored by [Satvik Ramaprasad](https://github.com/satu0king/), [Aboobacker MK](https://github.com/tachyons), [Shivansh Shrivastava](https://github.com/Shivansh2407) and [Aayush Gupta](https://github.com/aayush-05).
+
+4. [Abhishek Zade](https://github.com/ZadeAbhishek) will be working on [Compatibility for Touch Devices](https://github.com/CircuitVerse/CircuitVerse/wiki/GSoC%2721-Project-List#project-5---compatability-for-touch-devices) and will be mentored by [Satvik Ramaprasad](https://github.com/satu0king/), [Shreya Prasad](https://github.com/ShreyaPrasad1209/), [Nitin Singhal](https://github.com/nitin10s/) and [Samiran Konwar](http://github.com/abstrekt/).
+
+5. [Manjot Sindhu](https://github.com/manjotsidhu) will be working on [Interactive Book Integration for Mobile App](https://summerofcode.withgoogle.com/projects/#5929763765485568) and will be mentored under [Aboobacker MK](https://github.com/tachyons) and [Nitish Aggarwal](https://github.com/Nitish145).
+ 
+6. [Devansh Dixit](https://github.com/DevanshD3) will be working on [Project Recommendation System](https://github.com/CircuitVerse/CircuitVerse/wiki/GSoC%2721-Project-List#project-7---project-recommendation-system) and will be mentored under [Biswesh Mohapatra](https://github.com/biswesh456), [Satvik Ramaprasad](https://github.com/satu0king/), [Aboobacker MK](https://github.com/tachyons) and [Shivansh Shrivastava](https://github.com/Shivansh2407).
+
+
+Once again, congratulations! We are impressed by your accomplishments so far, and hope you are just as pleased to be part of our community of so many wonderful people!🎉 
```

Comment:
Do we need this here? As the targeted audience of the blog post is the wider community which includes teachers and students. I think we should use this space for telling what we achieved via gsoc before and what we are expecting this year from product's perspective.

Reply:
I feel that publicly recognising the student's achievements will motivate them to perform in a better way because they will know that their efforts are getting acknowledged! However, you have put a good point and this can be incorporated in a detailed manner in next blog 👍🏻

Reply:
the blog is very nice 🥰

Reply:
The date is in the past @ShreyaPrasad1209 ??
![image](https://user-images.githubusercontent.com/52851184/119777441-c82a4880-bee3-11eb-9b5c-5272fe9a850d.png)

Reply:
That's because I created it on that date. If I update it now then the date will be changed as well.